students/AntonBakaiev/hw-6+7/Task-1.js

```diff
@@ -0,0 +1,14 @@
+const enterNumber = prompt('Enter number');
+
+const initialNumber = () => {
+    if (enterNumber == 0) {
```

Comment:
Why you use non-strict equality operator here?
```suggestion
    if (enterNumber === 0) {
```

Reply:
виправив